oauth/resources/org.jaggeryjs.modules.oauth/src/main/java/org/jaggeryjs/modules/oauth/OAuthHostObject.java

```diff
@@ -68,14 +75,15 @@ public static Scriptable jsConstructor(Context cx, Object[] args, Function ctorO
 
                 oauthho.apiKey = providerConfig.getApi_key();
                 oauthho.apiSecret = providerConfig.getApi_secret();
+                oauthho.tokenEndpoint = providerConfig.getAccess_token_url();
 
                 if (providerConfig.getOAuth_version() == 1.0) {
-                	
+
                     if (providerConfig.getRequest_token_url() == null) {
                         throw new ScriptException("API configuration not specified");
                     }
-                    
-                	oauthho.oAuthVersion = OAuthVersion.OAUTH1;
+
+                    oauthho.oAuthVersion = OAuthVersion.OAUTH1;
                     GenericOAuth10aApi oauth10aApi = new GenericOAuth10aApi();
                     oauth10aApi.setAccessTokenEndpoint(providerConfig.getAccess_token_url());
                     oauth10aApi.setAuthorizationUrl(providerConfig.getAuthorization_url());
```

Comment:
Since methods of ProviderConfig are reflected from JS object. we did not changed any underscores in ProviderConfig class, so I am keeping the same patten. In future releases ProviderConfig class should be refactored.